### PR TITLE
first implementation

### PR DESCRIPTION
**Pull request by other91**
first implementation

### CODE REVIEW

File: ITRStartTask/StoriesList/Presenter/StoriesPresenter.swift [hunk 1]
```diff
@@ -18,6 +18,16 @@ class StoriesPresenter: StoriesPresenterProtocol {
         self.storiesDataProvider = storiesDataProvider
         self.storyItems = []
     }
+    func refresh() {
+        storiesDataProvider.getStoryItems(onSuccess: { [weak self] (storyItems) in
```
Comment (asleptsov-itr) [inline ITRStartTask/StoriesList/Presenter/StoriesPresenter.swift]: (storyItems) - remove braces

File: ITRStartTask/StoriesList/Views/StoriesViewController.swift [hunk 2]
```diff
@@ -38,13 +40,29 @@ class StoriesViewController: UIViewController, XibInitializable {
...
+    }
 }
 
 extension StoriesViewController: StoriesViewProtocol {
     func refreshStories() {
         self.tableView.reloadData()
     }
 
+    func endRefreshing() {
+        self.refreshControl.endRefreshing()
```
Comment (asleptsov-itr) [inline ITRStartTask/StoriesList/Views/StoriesViewController.swift]: self. is not needed here

File: ITRStartTask/StoryDetail/Views/StoryViewController.swift [hunk 3]
```diff
@@ -28,18 +28,18 @@ class StoryViewController: UIViewController, XibInitializable {
...
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         presenter.show()
     }
 
+    @objc func backPressed() {
+        self.navigationController?.popViewController(animated: true)
+    }
+
     private func replaceBackButton() {
```
Comment (asleptsov-itr) [inline ITRStartTask/StoryDetail/Views/StoryViewController.swift]: rename to configureBackButton

File: ITRStartTask/StoriesList/Presenter/StoriesPresenter.swift (hunk 1 above)
Comment (other91) [reply ITRStartTask/StoriesList/Presenter/StoriesPresenter.swift]: fixed

File: ITRStartTask/StoryDetail/Views/StoryViewController.swift (hunk 3 above)
Comment (other91) [reply ITRStartTask/StoryDetail/Views/StoryViewController.swift]: fixed

File: ITRStartTask/StoriesList/Views/StoriesViewController.swift (hunk 2 above)
Comment (other91) [reply ITRStartTask/StoriesList/Views/StoriesViewController.swift]: fixed

File: ITRStartTask/StoriesList/Views/StoriesViewController.swift [hunk 4]
```diff
@@ -80,7 +78,7 @@ extension StoriesViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCell(withIdentifier: StoriesViewController.storyCellId,
                                                        for: indexPath) as? StoryCell else {
-            return UITableViewCell(frame: .zero)
+                                                        return UITableViewCell(frame: .zero)
```
Comment (asleptsov-itr) [inline ITRStartTask/StoriesList/Views/StoriesViewController.swift]: remove spaces
UITableViewCell(frame: .zero) --> UITableViewCell()

Comment (other91) [reply ITRStartTask/StoriesList/Views/StoriesViewController.swift]: fixed

**Pull request merged**